modern/src/map/Map.js

```diff
@@ -93,6 +93,23 @@ const Map = ({ children }) => {
     };
   }, []);
 
+  useEffect(() => {
+    if (mapReady && Object.values(map.getStyle().sources).some(e => e.type === 'raster')) {
+      const onIdle = function () {
+        const zoom = Math.round(map.getZoom());
+        if (zoom !== map.getZoom()) {
+          map.zoomTo(zoom);
+        }
```

Comment:
Can this just be
```suggestion
        map.zoomTo(Math.round(map.getZoom()));
```

Reply:
this causes infinite loop since "idle" event will be fired after every zoom.